src/v0/destinations/tiktok_ads/data/TikTokTrackV2.json

```diff
@@ -126,5 +126,41 @@
     "destKey": "user.user_agent",
     "sourceKeys": ["properties.context.user.userAgent", "context.userAgent"],
     "required": false
+  },
+  {
+    "destKey": "user.first_name",
+    "sourceKeys": "firstName",
+    "metadata": {
+      "type": ["trim", "toLower", "hashToSha256"]
+    },
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "user.last_name",
+    "sourceKeys": "lastName",
+    "metadata": {
+      "type": ["trim", "toLower", "hashToSha256"]
+    },
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "user.city",
+    "sourceKeys": ["context.traits.city", "context.traits.address.city", "properties.city"]
+  },
+  {
+    "destKey": "user.country",
+    "sourceKeys": ["context.traits.country", "context.traits.address.country", "properties.country"]
+  },
+  {
+    "destKey": "user.state",
+    "sourceKeys": ["context.traits.address.state", "context.traits.state", "properties.state"]
```

Comment:
1. Can you check order and compare with other fields like city, country etc ?
2. Should we also add `context.traits.address.state` as one of the sourceKeys ?

Reply:
These are user properties so I guess we should be ok with these fields